src/main/java/optic_fusion1/mcantimalware/utils/Updater.java

```diff
@@ -0,0 +1,75 @@
+package optic_fusion1.mcantimalware.utils;

```

Comment:
You should use the logger for logging an exception and printing out information.

Reply:
Loggers being updated, might as well wait